### PR TITLE
Expose useNativeAnimations for Drawer navigator

### PR DESCRIPTION
**Pull request by jkimbo**
By enabling `useNativeAnimations` the drawer navigator is significantly quicker.

**Test plan (required)**

Option is opt in so shouldn't affect any current users.

### CODE REVIEW

**Comment by AndrewIngram** [comment]
Videos showing the different:

Before: https://www.youtube.com/watch?v=O3-YMhocBVw
After: https://www.youtube.com/watch?v=dSpF-WkP1Yc

File: src/navigators/DrawerNavigator.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const DefaultDrawerConfig = {
     Dimensions.get('window').width - (Platform.OS === 'android' ? 56 : 64),
   contentComponent: DrawerItems,
   drawerPosition: 'left',
+  useNativeAnimations: false,
```
Comment (kelset) [inline src/navigators/DrawerNavigator.js]: One thing: I think this should be set to true

Comment (jkimbo) [reply src/navigators/DrawerNavigator.js]: I defaulted to false because I didn't want to accidentally break someones app. Happy to change it though.

Comment (spencercarli) [reply src/navigators/DrawerNavigator.js]: Yeah I don't think it would have any negative ramifications to set this to default.

Comment (jkimbo) [reply src/navigators/DrawerNavigator.js]: Ok done!

**Comment by jkimbo** [comment]
@kelset thanks! Any eta on when it will be released?

**Comment by kelset** [comment]
Probably soon ;) We are working to get the roadmap to 1.0 out asap atm ;)

**Pull request merged**